roles/owncloud/tasks/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 
-- include: owncloud.yml tags=owncloud
\ No newline at end of file
+- include: owncloud.yml tags=owncloud
+  # https://github.com/sovereign/sovereign/issues/765
+  when: ansible_distribution_release != "jessie"
```

Comment:
What do you think about adding a „debug“ statement to let the user know what is going on? Something like „owncloud no longer supports Debian jessie“?

Reply:
Sure thing.